e2e-tests/cypress/integration/appellant-submission-your-details-v2-applicant-name.feature

```diff
@@ -0,0 +1,21 @@
+@smoketest
```

Comment:
what does `v1` and `v2` in the file names represent ? 

It seems like 'versions' is the obvious answer, but I've never seen this done before so am curious.

Reply:
I have used it to make clear version 1 of how we did feature files from the latest v2. I did not want to remove v1 just yet but I think we will do so in the future.